app/services/operations/search.rb

```diff
@@ -0,0 +1,30 @@
+module Operations
+  class Search
+    def initialize(search:, pagination:)
+      @filter = SearchFilter.new(search)
+      @pagination = Pagination.new(pagination)
+      @scope = CrimeApplication
+    end
+
+    def call
+      query
```

Comment:
How is query i.e. `CrimeApplication` able to recieve `page` here? Is it due to the presenters on L19 in `app/api/datastore/v2/searches.rb` little confused by this.

Reply:
I'll try to make this clearer in the next pr.